server/api/harvestingrequests.js

```diff
@@ -298,7 +298,7 @@ const convertRequest = (req, res) => {
                 license: harvestReq.license,
                 harvestReqID: harvestReq._id
             };
-            if (harvestReq.url) projectData.resourceURL = harvestReq.url;
+            if (harvestReq.url) projectData.license.sourceURL = harvestReq.url;
```

Comment:
can we double check that `projectData.license` will always have been initialized